io/io/CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ if (uring)
 endif ()
 
 ROOT_LINKER_LIBRARY(RIO
-  src/RRawFile.cxx
   ${rawfile_local_sources}
```

Comment:
the `${rawfile_local_sources}` and `${rawfile_local_headers}` need to move to iolite, too

Reply:
Should we move as well RIoUring.hxx?

Reply:
@jblomer actually I checked and it has dependency on `uring` library that is a bit complex to handle. Can move it in a second iteration please?